core/trino-spi/src/test/java/io/trino/spi/predicate/TestTupleDomain.java

```diff
@@ -657,8 +657,8 @@ public void testTransformFailsWithNonUniqueMapping()
     private boolean overlaps(Map<ColumnHandle, Domain> domains1, Map<ColumnHandle, Domain> domains2)
```

Comment:
Please do not use `.` and the end of commit message title. Please see: https://github.com/trinodb/trino/blob/master/DEVELOPMENT.md#format-git-commit-messages